doc/developer/requirements.md

```diff
@@ -127,3 +122,33 @@ You can also run any cozy-stack command with the `yarn cozy-stack` script, e.g.:
 ```
 yarn cozy-stack apps install --domain cozy.tools:8080 drive 'git://github.com/cozy/cozy-drive.git#build-drive'
 ```
+
+## cozy-desktop
+
+### Create a transifex API token
+
+- Create an account on http://www.transifex.com
+- Join the Cozy team
+- Get your API token from the account settings page
+
+### Install dependencies
+
+```
+yarn install
+```
+
+### Build everything
+
+```
+yarn build
+```
+
+The requested token is your transifex API token.
```

Comment:
Those are already in `setup.md`.

Initially we only had `doc/developer/setup.md`.
Then @enguerran extracted `doc/developer/requirements.md` from it.
Now we're moving yarn install/build sections from `setup.md` to `requirements.md`...
So I'm a bit confused :slightly_smiling_face:

What do we expect from this `requirements.md` file?
Should it list everything required to build the app? (what I was expected, but maybe I'm wrong)
In which case, `yarn install` could be considered a requirement, but not `yarn build` since it's the actual build?

What else from `setup.md` should or should not go into `requirements.md`?
Should we turn `setup.md` into a `README.md` including links to other docs (or just single commands when possible) in the most logical order? (sounds good to me)

Please note building could deserve its own `build.md` doc, explaining how to build Elm/Stylus & update transifex translations at once (this is `yarn build`) and also how to build the final packages (`yarn dist` and its various options). But this one is definitely out of the scope of this PR.

Reply:
I added the yarn commands because I didn't see anything about checking the build is working. I'm not sure separating requirements from the rest of the setup is best but this was probably done for a reason.

Maybe I could add a link back to `setup.md` from the requirements instead?